src/dialects/db2/query-generator.js

```diff
@@ -857,36 +857,40 @@ export class Db2QueryGenerator extends Db2QueryGeneratorTypeScript {
     return '';
   }
 
+  // TODO: migrate to Db2QueryGeneratorTypeScript
+  // TODO: how to deal with this void return? since the abstract method returns string
```

Comment:
I tried returning `string | void` in the abstract method, but eslint complains due to https://typescript-eslint.io/rules/no-invalid-void-type/

Reply:
Something that doesn't support setting the isolation level should throw if someone tries to use that method

Reply:
Oh yeah, that should also works. Forgot about that option